WcaOnRails/app/views/static_pages/about.html.erb

```diff
@@ -15,14 +15,15 @@
   <p><%= t('about.structure.board_html') %></p>
 
   <ul>
-    <% @board_members.order(:name).each do |board_member| %>
+    <% User.where(board_member?).order(:name).each do |board_member| %>
```

Comment:
This is very interesting to me. Does this actually work? If it does work, is this causing us to look at every single user in our database and run `board_member?` on each of them? If so, that will probably be very slow!

I think it would be better to look up the team, and then iterate over it's members.

Reply:
Actually I didn't test it yet. I was not in my computer, so I just commited to save it and continue later.
Your approach is indeed better.